src/sentry/incidents/models.py

```diff
@@ -539,8 +539,9 @@ class Type(Enum):
         PAGERDUTY = 1
         SLACK = 2
         MSTEAMS = 3
+        INTEGRATION = 4
```

Comment:
@mgaeta Similar to what Meredith said earlier, calling it `INTEGRATION` here is confusing since Slack, Pagerduty, and Teams are all what we call integrations.